### PR TITLE
HTML issue#3

### LINKED ISSUE #3: HTML
# HTML Qualification

| **Task:** | Demonstrate the ability to build a web page with nested HTML. |
| - | - |
| **Conditions:** | This Github repository is downloaded locally and loaded into vs-code. |
| **Standards:** | In a Pull Request (PR) with a Netlify preview enabled create a HTML webpage with the following: <ul><li>three different types of headings</li><li>two paragraphs with three different kinds of inline elements</li><li>A `br` and a `hr`</li><li>A header and footer</li><li>Two different `section`s</li><li>An unordered list of three items</li><li>An anchor tag with a link</li><li>A Form with a `textarea`, three types of `input`s` and a button </li><li>A table with three columns and three rows with a header, body, and footer</li></ul> |

## Notes:
- Create a Pull Request (PR) to this Github Repo and assign the `web-qualifed` team. 

## References
- [Deploy Netlify](https://www.netlify.com/blog/2016/09/29/a-step-by-step-guide-deploying-on-netlify/)
- [Getting to know HTML](https://learn.shayhowe.com/html-css/getting-to-know-html/)
- [Building Forms](https://learn.shayhowe.com/html-css/building-forms/)

### PR DESCRIPTION
**Pull request by JVSONV**
Resolves #3 

Netlify address [HTML issue#3](https://melodious-moonbeam-46a70d.netlify.app/)

### CODE REVIEW

File: issueWork/HTML__issue/index.html [hunk 1]
```diff
@@ -0,0 +1,140 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script defer>
+      document.querySelector();
```
Comment (axecopfire) [inline issueWork/HTML__issue/index.html:8]: This is unneeded for now

**Review by axecopfire — CHANGES_REQUESTED** [review]
Remove javascript, and add deploy preview

File: issueWork/HTML__issue/index.html (hunk 1 above)
Comment (JVSONV) [reply issueWork/HTML__issue/index.html:8]: Oops I forgot it was in there. After I finished it I was just going to make this document meet all the requirement for both issues 3 && 4 and give the form submit a funny little browser alert. In the end I decided not to last night and went to dinner with my wife. I just forgot to remove it. Sorry!

Comment (axecopfire) [reply issueWork/HTML__issue/index.html:8]: No need to apologize, ur good

File: issueWork/HTML__issue/index.html [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+          </label>
+          <fieldset>
+            <legend>Please select one:</legend>
+            <div>
+              <input
+                id="chrome"
+                type="radio"
+                name="unorderedList"
+                value="Chrome_Inspects"
+              /><label for="chrome">Chrome Inspects</label>
```
[the remark this answers is not in the document]
Comment (JVSONV) [reply issueWork/HTML__issue/index.html:76]: Apparently Prettier cannot auto format HTML [cannot auto format HTML](https://github.com/prettier/prettier-vscode/issues/2478) even though it is the default this is an extra setting and prompt that will only appear when highlighting  a snippet of code and selecting format selection. At that point you be notified that this is the case and be provided a working alternative for HTML documents. Who know?

[the remark this answers is not in the document]
Comment (axecopfire) [reply issueWork/HTML__issue/index.html:76]: I was able to use Prettier to format your code. But it's not as impactful as I thought. I added a piece of feedback to the doc to add an automated check to PRs to check whether the code is formatted or not. I probably shouldn't be checking for formatting.

**Review by axecopfire — APPROVED** [review]

**Pull request merged**